draft/2022-09-14-this-week-in-rust.md

```diff
@@ -153,27 +140,28 @@ Rusty Events between 2022-09-14 - 2022-10-12 🦀
     * [**Going full stack on Rust**](https://www.meetup.com/dutch-rust-meetup/events/286727064/)
 * 2022-10-02 | Florence, IT + Virtual | [RustLab](https://rustlab.it/)
     * [**RustLab Conference 2022 (Oct 2-4)**](https://rustlab.it/schedule/)
+* 2022-10-03 | Stockholm, SE | [Stockholm Rust](https://www.meetup.com/Stockholm-Rust/)
+    * [**Rust Meetup @Microsoft Reactor**](https://www.meetup.com/Stockholm-Rust/events/288453360/)
+* 2022-10-12 | Berlin, DE | [Rust Berlin](https://www.meetup.com/rust-berlin/)
+    * [**Rust and Tell - EuroRust B-Sides**](https://www.meetup.com/rust-berlin/events/288175448/)
 
 ### North America
 
-* 2022-09-08 | Columbus, OH, US| [Columbus Rust Society](https://www.meetup.com/columbus-rs/)
-    * [**Monthly Meeting**](https://www.meetup.com/columbus-rs/events/dpkhgrydcmblb/)
 * 2022-09-14 | Austin, TX, US | [Rust ATX](https://www.meetup.com/rust-atx/)
     * [**Rust Lunch**](https://www.meetup.com/rust-atx/events/288287206/)
 * 2022-09-20 | San Francisco, CA, US | [San Francisco Rust Study Group](https://www.meetup.com/san-francisco-rust-study-group/)
     * [**Rust Hacking in Person**](https://www.meetup.com/san-francisco-rust-study-group/events/wjkjssydcmbbc/)
 * 2022-09-22 | Lehi, UT, US | [Utah Rust](https://www.meetup.com/utah-rust/)
     * [**Game Prototyping with Rusty Engine with Nathan Stocks and Food!**](https://www.meetup.com/utah-rust/events/rvpgxsydcmbmc/)
+* 2022-09-29 | Ciudad de México, MX | [Rust MX](https://www.meetup.com/rust-mx/)
+    * [**Nuestra RustMX comunidad tiene página**](https://www.meetup.com/rust-mx/events/288388973/)
```

Comment:
May wish to add:
10/12/22
Microsoft Reactor SF
Getting Started with Rust: Building Rust Projects
https://www.meetup.com/microsoft-reactor-san-francisco/events/288475796

Reply:
Done!